.github/workflows/test.yml

```diff
@@ -0,0 +1,56 @@
+name: Test Postgres
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+    paths-ignore:
+      - "*.md"
+
+jobs:
+  build:
+    name: Test P${{ matrix.python-version }}, PSQL${{matrix.postgres-version}}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.7", "3.9"]
+        postgres-version: [10, 11, 12]
+        include:
+          - tox_test: "test_py37"
+            python-version: "3.7"
+          - tox_test: "test_py39"
+            python-version: "3.9"
+
+    services:
+      postgres:
+        # we need postgres with plpython extension postgres base docker hub image did not have that
+        # I use this one
+        image: thorinschiffer/postgres-plpython:${{ matrix.postgres-version }}
+        env:
+          POSTGRES_PASSWORD: postgres
+          # GreenplumPython use gpadmin as db name
+          POSTGRES_DB: gpadmin
+        ports:
+        - 5432:5432
+  
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+        cache: pip
+    - name: Install Dependencies
+      run: |
+        pip install tox==3.25.0
+    - name: Create plpython3u language and environment
+      run: |
+        psql postgres://postgres:postgres@localhost/gpadmin -c "CREATE LANGUAGE plpython3u;"
+    - name: Run Tests
+      run: |
+        export POSTGRES_PASSWORD=postgres
```

Comment:
I think if you use  `PGPASSWORD` and `PGUSER`, you don't even need to change the python files.

Reply:
```python
    db = gp.database(
        host="localhost",
        dbname="gpadmin",
        user=environ.get("POSTGRES_USER"),
        password=environ.get("POSTGRES_PASSWORD"),
```
can the user and password just pass?

Reply:
let me try.

Reply:
Agree to use PGUSER and PGPASSWORD

Reply:
database name PGDB?